Makefile

```diff
@@ -23,7 +23,7 @@ BUILD_DATE_VAR := github.com/openservicemesh/osm/pkg/version.BuildDate
 BUILD_VERSION_VAR := github.com/openservicemesh/osm/pkg/version.Version
 BUILD_GITCOMMIT_VAR := github.com/openservicemesh/osm/pkg/version.GitCommit
 DOCKER_GO_VERSION = 1.17
-DOCKER_BUILDX_PLATFORM ?= linux/amd64
+DOCKER_BUILDX_PLATFORM ?= linux/arm64
```

Comment:
@nojnhuh  opinion on what we should do about this?

Reply:
Please keep the default as before `linux/amd64`.

Reply:
Hi @shashankram  - thanks for your suggestion! If I keep it as the default of `linux/amd64` it negates the reasoning of this PR - do you think the suggestion from @nojnhuh would be a reasonable middle ground?
Thanks!

Reply:
FWIW, if we decide to stick with linux/amd64 hardocded as the default, you could set this as an environment variable to be whatever default you want and I don't think you'd need to do anything extra for that to be picked up.

Reply:
This can be overriden as an argument to `make`, but I see @nojnhuh's suggestion and agree with it.

Reply:
Thanks everyone!